package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "start": "./entry.sh",
-    "build": "coffee -c src",
+    "build": "coffee -c src && webpack -p",
```

Comment:
`-p`? Can you use a long arg if there is one, as I have no idea off-hand what `-p` is